source/unified-test-format/tests/invalid/clientEncryptionOpts-tlsOptions_not_supported.yml

```diff
@@ -0,0 +1,19 @@
+description: clientEncryptionOpts-tlsOptions_not_supported
```

Comment:
This is technically redundant given `clientEncryptionOpts-invalid_property`, which I suggested renaming to `clientEncryptionOpts-additionalProperties`; however, I think it's worth keeping since `tlsOptions` does appear in the CSFLE spec.

Feel free to capture that in a comment above the description.

```suggestion
# Note: this covers the same rule as clientEncryptionOpts-additionalProperties,
# but exists independently since tlsOptions is defined in the CSFLE spec.
description: clientEncryptionOpts-tlsOptions_not_supported
```